caddy/caddy.go

```diff
@@ -104,6 +100,48 @@ func (f *FrankenPHPApp) Provision(ctx caddy.Context) error {
 	return nil
 }
 
+func (f *FrankenPHPApp) generateUniqueModuleWorkerName(filepath string) string {
+	var i uint
+	filepath, _ = fastabs.FastAbs(filepath)
+	name := "m#" + filepath
+
+outer:
+	for {
+		for _, wc := range f.Workers {
+			if wc.Name == name {
+				name = fmt.Sprintf("m#%s_%d", filepath, i)
+				i++
+
+				continue outer
+			}
+		}
+
+		return name
+	}
+}
+
+func (f *FrankenPHPApp) AddModuleWorkers(workers ...workerConfig) ([]workerConfig, error) {
```

Comment:
This function could be unexported, right?

```suggestion
func (f *FrankenPHPApp) addModuleWorkers(workers ...workerConfig) ([]workerConfig, error) {
```

Reply:
yep